### PR TITLE
Prevent sizing down the mini-buffer window too quickly

### PR DESCRIPTION
**Pull request by QiangF**
This is an ugly work around for https://github.com/kiennq/emacs-mini-modeline/issues/10.
Maybe there is a better way.

### CODE REVIEW

File: mini-modeline.el [hunk 1]
```diff
@@ -177,14 +178,27 @@ When ARG is:
...
+                              (height-increase (- (cdr mini-modeline--cache)
+                                                  (window-height (minibuffer-window mini-modeline-frame)))))
+                          (when (or (> height-increase 0)
+                                    ;; this is to prevent window flashing for consecutive multi-line message
+                                    (>= (float-time (time-since mini-modeline--last-change-size))
+                                        mini-modeline-echo-duration))
+                            (window-resize (minibuffer-window mini-modeline-frame)
+                                           height-increase)
+                            (setq mini-modeline--last-change-size (current-time)))))
+                      (insert (car mini-modeline--cache))))))))
```
Comment (kiennq) [inline mini-modeline.el]: This seems not properly indented.

File: mini-modeline.el [hunk 2]
```diff
@@ -177,14 +178,27 @@ When ARG is:
                   ;; write to minibuffer
                   (erase-buffer)
                   (when mini-modeline--cache
-                    (if (> (cdr mini-modeline--cache) 1)
-                        (window-resize (minibuffer-window mini-modeline-frame)
-                                       (- (cdr mini-modeline--cache)
-                                          (window-height (minibuffer-window mini-modeline-frame)))))
-                    (insert (car mini-modeline--cache))))))))
+                    (if (>= (cdr mini-modeline--cache) 1)
```
Comment (kiennq) [inline mini-modeline.el]: No need to check this `(cdr mini-modeline--cache)` is always >= 1

File: mini-modeline.el [hunk 3]
```diff
@@ -177,14 +178,27 @@ When ARG is:
...
+                                    (>= (float-time (time-since mini-modeline--last-change-size))
+                                        mini-modeline-echo-duration))
+                            (window-resize (minibuffer-window mini-modeline-frame)
+                                           height-increase)
+                            (setq mini-modeline--last-change-size (current-time)))))
+                      (insert (car mini-modeline--cache))))))))
       ((error debug)
        (mini-modeline--log "mini-modeline: %s\n" err)))))
 
+(defun mini-modeline-minibuffer-setup ()
```
Comment (kiennq) [inline mini-modeline.el]: Please name this and the function bellow `mini-modeline--....`

**Comment by kiennq** [comment]
Thanks for your PR. This LGTM. Can merge after your address my comments.

File: mini-modeline.el (hunk 1 above)
Comment (QiangF) [reply mini-modeline.el]: I use Lispy for the indentation, I really don't have an idea on what it should be like.

**Comment by kiennq** [comment]
LGTM now. Merged

**Pull request merged**